test/e2e/apps/job.go

```diff
@@ -361,6 +368,172 @@ var _ = SIGDescribe("Job", func() {
 			framework.ExpectEqual(pod.Status.Phase, v1.PodFailed)
 		}
 	})
+
+	ginkgo.It("should test the lifecycle of a job", func() {
```

Comment:
Can you add a summary of the test as a comment above?

Reply:
Added a description for the test. I've used the [test comment metadata](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-test-comment-metadata) format required for conformance tests. Saving some time for when the test is [promoted to conformance](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#promoting-tests-to-conformance) in the near future.

Reply:
Oh, same problem here

```suggestion
	ginkgo.It("should manage the lifecycle of a job", func() {
```

`It` refers to the API and controller, Job in this case, not the test.

Reply:
Fixed